dds/DCPS/security/framework/SecurityRegistry.cpp

```diff
@@ -444,10 +447,12 @@ SecurityRegistry::get_plugin_inst(const OPENDDS_STRING& plugin_name)
     // Try to find it again
     if (find(registered_plugins_, plugin_name, plugin_inst) != 0) {
 #endif
-      ACE_ERROR((LM_ERROR,
-                 ACE_TEXT("(%P|%t) SecurityRegistry::create_inst: ")
-                 ACE_TEXT("plugin_type=%C is not registered.\n"),
-                 plugin_name.c_str()));
+      if (log_error) {
```

Comment:
Could this just be removed?  The error can be logged by the caller if needed (and looks like it already is).

Reply:
I didn't notice, yes I will remove it.